policyengine_us/variables/gov/states/md/tax/income/agi/subtractions/md_two_income_subtraction.py

```diff
@@ -0,0 +1,61 @@
+from policyengine_us.model_api import *
+
+
+class md_two_income_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD two-income married couple subtraction from AGI"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.marylandtaxes.gov/forms/21_forms/Resident_Booklet.pdf#page=16"
+    defined_for = StateCode.MD
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        # compute head and spouse US AGI portions using irs_gross_income shares
+        us_agi = tax_unit("adjusted_gross_income", period)
+        gross_income = person("irs_gross_income", period)
+        is_head = person("is_tax_unit_head", period) == 1
```

Comment:
Could you try without the `== 1`? It should work since these are bools.

Reply:
OK.  I've simplified the `is_head` and `is_spouse` statements.